### ISSUE TITLE
Sometimes the driver seems to die with device error -17 and only recovers via a reboot

### ISSUE THREAD

**Issue by oviano** [oneapi-src/oneVPL#81]
This is Windows 11 Pro, 3802 driver, 2022.2.5 oneVPL release.

For example, I'm transcoding with FFmpeg decoding with hevc_qsv and transcoding to av1_qsv and it's working fine. I then leave the PC idle for an hour or two, come back and run the exact same command and this time I get the dreaded device error -17 from the tool when trying to decode.

It happens perhaps once or twice a day, apparently randomly. I.e. it is not preceded by any other errors or unexpected behaviour.

I have tried disabliing/renabling the device in control panel, but it seems the only cure is to reboot the PC.

Firstly, is this a known issue?

Secondly, is there another approach I can take from an API level that would "reset" the state of the device?

**Comment by rupakroyintel** [comment]
@oviano  This type of issue can be caused by the additional allocation of hardware buffers by FFmpeg that weren't freed in time. Could you please provide the command line that you are using and the screenshot of the error that you got? Thanks.

**Comment by oviano** [comment]
Next time it happens, I'll post back with more details.

From memory, it isn't one particular command, and it can work for hours and then suddenly produce this error.

**Comment by rupakroyintel** [comment]
@oviano Yes, a memory dump might be useful. Even if you do not know exactly which command line is the problem, some example command lines could give us some clues. 
Meanwhile, you can try to update the driver.  [31.0.101.3959 ](https://www.intel.com/content/www/us/en/download/729157/intel-arc-graphics-windows-dch-driver-beta.html) is the latest beta driver for Intel® Arc™ A-Series Graphics.

**Comment by rupakroyintel** [comment]
@oviano Thanks for sharing the details with us. We have made the relevant team aware of this issue. We will get back to you soon.

**Comment by ma3uk** [comment]
I have a similar problem. The problem can arise simply by starting a new encoding process immediately after the previous one, the error can be either -17 or -16

**Comment by mav-intel** [comment]
Transferring to the appropriate project
